smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestResponseGenerator.kt

```diff
@@ -67,6 +57,26 @@ open class HttpProtocolUnitTestResponseGenerator protected constructor(builder:
         writer.write("}")
     }
 
+    private fun renderBuildHttpResponseParams(test: HttpResponseTestCase) {
+        val params = mutableListOf<String>()
+        params.add("code: ${test.code}")
+        if (test.headers.isNotEmpty()) {
+            var headers = mutableListOf<String>()
+            for (hdr in test.headers.entries) {
+                headers.add("    \"${hdr.key}\": \"${hdr.value}\"")
+            }
+            val headersString = "headers: [\n${headers.joinToString(",\n")}\n]"
+            params.add(headersString)
+        }
+        test.body.ifPresent { body ->
+            if (body.isNotBlank() && body.isNotEmpty()) {
+                params.add("content: HttpBody.stream(ByteStream.from(data: \"\"\"\n${body.replace(".000", "")}\n\"\"\".data(using: .utf8)!))")
+            }
+        }
+        val paramsWithDelimiter = params.joinToString(",\n")
+        writer.write(paramsWithDelimiter)
```

Comment:
nit(up to you if you wanna leave/change)
writer.write(params.joinToString(",\n"))

Reply:
changed